pkgs/top-level/python-packages.nix

```diff
@@ -2546,6 +2546,8 @@ in {
 
   fido2 = callPackage ../development/python-modules/fido2 { };
 
+  file_read_backwards = callPackage ../development/python-modules/file_read_backwards { };
```

Comment:
I think you should also use `file-read-backwards` here. Also maybe rename the filename (it's always nice if the name of the file matches the attr name).